runescape-api/src/main/java/net/runelite/rs/api/RSClient.java

```diff
@@ -972,4 +972,8 @@
 	@Import("healthbarCache")
 	@Override
 	RSNodeCache getHealthBarCache();
+	
+	@Import("renderSelf")
+	void toggleRenderSelf();
```

Comment:
this needs a client transform using obfuscated fields, not sure if @zeruth has that autoupdating with fields yet but currently (179 gamepack) its `boolean client.jb` and the method body is `client.jb = !client.jb;`

Reply:
@zeruth? if not, I can revert that for now.